packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -224,15 +226,13 @@ const node = new appmesh.VirtualNode(this, 'node', {
 
 const virtualService = new appmesh.VirtualService(this, 'service-1', {
   virtualServiceProvider: appmesh.VirtualServiceProvider.virtualRouter(router),
-  virtualServiceName: 'service1.domain.local',
+  virtualServiceName,
 });
-
-node.addBackend(appmesh.Backend.virtualService(virtualService));
 ```
 
 The `listeners` property can be left blank and added later with the `node.addListener()` method. The `serviceDiscovery` property must be specified when specifying a listener.
 
-The `backends` property can be added with `node.addBackend()`. In the example, we define a virtual service and add it to the virtual node to allow egress traffic to other nodes.
```

Comment:
Similar to above feedback, let's keep this line since customer can still perform `node.addBacked()`.